tests/GrabyFunctionalTest.php

```diff
@@ -340,4 +340,36 @@ public function testCookie(): void
         // if the cookie wasn't taking into account, it'll be "NPR Choice page"
         $this->assertSame('Michael Flynn\'s Contradictory Line On Russia', $res['title']);
     }
+
+    public function testSaveXmlUnknownEncoding(): void
+    {
+        $graby = new Graby([
+            'debug' => true,
+            'extractor' => [
+                'config_builder' => [
+                    'site_config' => [__DIR__ . '/fixtures/site_config'],
+                ],
+            ],
+        ]);
+        $res = $graby->fetchContent('http://motherjones.com/politics/2012/02/mac-mcclelland-free-online-shipping-warehouses-labor');
```

Comment:
I wonder if we could store the response for mocking purposes in case the site ever gets down.

Reply:
I started that for few cases but it's really boring to perform 😫
https://github.com/j0k3r/graby/tree/master/tests/fixtures/content

Reply:
I guess I could try making an automatic rule for https://github.com/rectorphp/rector.

Reply:
You mean a rule that run the test suite and save content fetched in file?

Reply:
Opened #282.